test/src/test/java/hudson/model/FingerprintCleanupThreadTest.java

```diff
@@ -52,6 +54,9 @@
     private Path tempDirectory;
     private Path fpFile;
 
+    @Rule
+    public JenkinsRule j = new JenkinsRule();
```

Comment:
better to name `jenkinsRule`

Reply:
@afalko I would agree with naming it jenkinsRule, but it seems like this is pretty common (standard-ish) in the codebase [[src](https://www.jenkins.io/doc/developer/testing/#providing-environment-variables)] . 
Let me know your thoughts, I have no issues in changing :)

Reply:
Sheep will other sheep off a cliff :p. Its not a big deal in this case, and is a non-blocking suggestion.

Reply:
I have tended to use `r` FWIW.

Reply:
Thanks @afalko I have updated it to `jenkinsRule` :)